x-pack/metricbeat/module/aws/billing/billing.go

```diff
@@ -444,8 +444,7 @@ func (m *MetricSet) getAccountName(svc *organizations.Client) map[string]string
 	for paginator.HasMorePages() {
 		page, err := paginator.NextPage(context.Background())
 		if err != nil {
-			//TODO continue or return with error? Probably is return but...
-			continue
+			return accounts
```

Comment:
do we not want to surface the error to the caller so they are aware the return value may not contain the full list of accounts?

Reply:
indeed we could add a log entry at warn level
when I tested the code the error I had was that I have no permission for the API call

"funny" enough that the paginator for an API call that I don't have permission to call tell us that "there are more pages"